content/en/blog/how-to-use-fission-postgresql-serverless-database/index.md

```diff
@@ -0,0 +1,170 @@
+categories = ["Tutorial","Application","Python","Database"]
+type = "blog"
++++
+
+Software development and deployment methods have evolved greatly over the years. From the times when system admins used to setup servers to run software applications to today when developers are not concerned about how the server is provisioned. Containers and container orchestration tools like Kubernetes help developers deploy serverless applications quickly. Similarly serverless functions like [Fission](https://fission.io) help you focus on the code rather than the infrastructure. 
+
+In this blog post, I'll show you how to use **Fission with PostgreSQL serverless database**.
+
+{{< imgproc fission-postgreSQL-serverless-database.png Fit "1000X1000" >}}
+Fission with PostgreSQL Serverless Database
+{{< /imgproc >}}
+
+## Voting App using Fission & PostgreSQL Serverless Database
+
+In this example, we will create a basic voting application that allows users to vote for an option and view the results. This application is written in **Python** and uses **PostgreSQL** database to store the results. By the end of this blog post you would have learnt how to use Fission with PostgreSQL serverless database. You can then extend this application to use Fission with other relational and non relational databases.
+
+So let's get started!
+
+{{< imgproc  myvotingapp-app.png Fit "1000X1000" >}}
+Voting App using Fission & PostgreSQL Serverless Database
+{{< /imgproc >}}
+
+## Pre Requisites
+
+### Installing PostgreSQL
+
+Before you start working on this demo, you need to ensure that you have Fission installed and running on your system. You can refer to our [Fission Installation](https://fission.io/docs/installation/) guide for more. Further, this example uses PostgreSQL hence you must also have a PostgreSQL instance in place. You can either run it on your system locally, in a docker container or your Kubernetes cluster.
+
+Below are few guides you can refer to for installing PostgreSQL:
+
+- Installing PostgreSQL on your local system - <a href="https://www.postgresql.org/download/" target="_blank">Postgresql.org</a>
+- PostgreSQL Docker container - <a href="https://hub.docker.com/_/postgres" target="_blank">Docker Hub</a>
+- PostgreSQL HELM charts - <a href="https://bitnami.com/stack/postgresql/helm" target="_blank">Bitnami</a> - *used in this example*
+
+<br>
+
+> *Note: If you have installed it using Helm Charts, please ensure to note down the service url and port number. This will be used to connect to the database from the function. You can get it by running  `helm status postgresql` in your terminal.*
+
+{{< imgproc myvotingapp-postgres-helm.png Fit "1000X1000" >}}
+Helm PostgreSQL Status
+{{< /imgproc >}}
+
+#### Database and Table Setup
+
+For this application, we have created a database named `votedb` and a table named `votebank`. You can use the following query to create the table:
+
+```  
```

Comment:
we can mention language type as `sql` here.

Reply:
updated